tests/test_api/fixtures/__init__.py

```diff
@@ -516,3 +516,52 @@ def test_pagination(self):
                     self.assertEqual(count_cbp, count_obp, "OBP<>CBP")
             finally:
                 self.destroy_objects()
+
+class IncrementalPaginationTestCase(ModifiableApiTestCase):
+
+    pagination_limit = 100
+    skip_obp = False
+
+    def create_objects(self):
+        """ Implement this method to guarantee a minimum amount of objects """
+        pass
+
+    def destroy_objects(self):
+        """ Implement this method if destroy_many is not applicable """
+        pass
+
+    def count_objects_by_pagination_type(self, time_based=False, per_page=None):
+        count = 0
+        generator = self.api(start_time=0, paginate_by_time=True, per_page=per_page) if time_based is True else self.api(start_time=0, paginate_by_time=False, per_page=per_page)
```

Comment:
can I suggest some new lines for this conditional?

Reply:
```suggestion
        if time_based is True:
            generator = self.api(
                start_time=0,
                paginate_by_time=True,
                per_page=per_page
            )
        else:
            generator = self.api(
                start_time=0,
                paginate_by_time=False,
                per_page=per_page
            )
```

I believe this to be the equivalent format.